sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -31,6 +35,23 @@ public SmsClient createSmsClientUsingAzureKeyCredential() {
         return smsClient;
     }
 
+    public SmsAsyncClient createSmsAsyncClientUsingAzureKeyCredential() {
```

Comment:
I dont think we need one for both async and sync clients. 
I would suggest picking one of the two to reduce duplication

Reply:
just let this one as async client to show developer they can create an async client